src/homework2.sql

```diff
@@ -0,0 +1,70 @@
+USE baseball;
+
+DROP TABLE IF EXISTS batter_game_stat;
+
+CREATE TABLE batter_game_stat AS
```

Comment:
I like the idea of creating this table for the stats, since the original tables have many columns, this ways makes it easier to read with only the columns needed for the assignment.